CHANGELOG.md

```diff
@@ -57,6 +57,40 @@ Swift 5.0
 Swift 4.2
 ---------
 
+* [SE-0185][]
+
+  Protocol conformances can now be synthesized in extensions in the same file as
+  the type definition, allowing automatic synthesis of conditional conformances
+  to `Hashable`, `Equatable` and `Codable` (both `Encodable` and
+  `Decodable`). For instance,
+
+  ```swift
+  struct Generic<Param> {
+      var property: Param
+  }
+  extension Generic: Equatable where Param: Equatable {}
+  ```
+
+  Code that wants to be as precise as possible should generally not
+  conditionally conform to `Codable` directly, but rather its two constituent
+  protocols, or else one can only (for instance) decode a `Generic<Param>` if
```

Comment:
"two constituent protocols" -> "two constituent protocols Foo and Bar" 

where Foo and Bar are the constituent protocols...